### PR TITLE
Azure customer self managed

### PR DESCRIPTION
**Pull request by flamarion**
## Description

Reorganize the W&B Server documentation and adds more documents

## Ticket

Does this PR fix an existing issue? If yes, provide a link to the ticket here:

## Checklist

Check if your PR fulfills the following requirements. Put an `X` in the boxes that apply.

- [x] Files I edited were previewed on my local development server with `yarn start`. My changes did not break the local preview.
- [x] Build (`yarn build:prod`) was run locally and successfully without errors or warnings.
- [x] I merged the latest changes from `main` into my feature branch before submitting this PR.

### CODE REVIEW

**Comment by flamarion** [comment]
Please, review (especially the grammar 😄 ) 

[Mock](https://user-images.githubusercontent.com/29267749/234309416-403d566e-171d-40bf-8076-c6907491ef3a.mov)

File: docs/guides/hosting/installation.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+## W&B Production and Develpoment
+
+In this page you will find all available installation methods of W&B server.
+
+The production installation types available are the following:
+
+- [AWS](/guides/hosting/installation/aws-tf)
+- [Azure](/guides/hosting/installation/azure-tf)
+- [GCP](/guides/hosting/installation/gcp-tf)
+- [Bare Betal](/guides/hosting/installation/bare-metal)
```
Comment (venky-wandb) [inline docs/guides/hosting/installation.md]:
```suggestion
- [Bare Metal](/guides/hosting/installation/bare-metal)
```

File: docs/guides/hosting/configuration.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+---
+slug: /guides/hosting/configuration
```
Comment (yogeshg) [inline docs/guides/hosting/configuration.md:2]: **suggestion (if minor, non-blocking):** [<>/guides/hosting](https://docs.wandb.ai/guides/hosting) is kinda odd url for product called W&B Server , we should change that if y'all agree.

**Comment by ngrayluna** [comment]
Closing this PR and working off of https://github.com/wandb/docodile/pull/278

**Pull request closed without merging**